docs/_docs/Applicants/Application-process.md

```diff
@@ -73,101 +73,85 @@ or at any time during the course, I will be expelled from the Academy.
 
 ## Application steps and important dates
 
-You can check the official calendar for Batch 6 [here](https://calendar.google.com/calendar/embed?src=c_jm13ojnma9kcfc3ehr22qvkq9c%40group.calendar.google.com&ctz=Europe%2FLisbon).
-
-### Wave 1
-
-| Activity                          | Date/Deadline                                    |
-|-----------------------------------|--------------------------------------------------|
-| Sign-up on the Admission Portal   | 2022-07-08 00h00 GMT+1 to 2022-07-10  23h59 GMT+1 |
-| Scholarship applications          | 2022-07-08 00h00 GMT+1 to 2022-07-10  23h59 GMT+1 |
-| Test & SLUs                       | 2022-07-11 00h00 GMT+1 to 2022-07-24 23h59 GMT+1 |
-| Candidate Selection               | 2022-07-25                                       |
-| Payments & scholarship interviews | 2022-07-26 00h00 GMT+1 to 2022-07-31 23h59 GMT+1 |
-| Admissions closed                 | 2022-08-01 23h59 GMT+1                           |
-
-If you complete the signup in the portal and confirm your email address, you'll always receive an email with the outcome of your application. This email should be sent between the 25th of July and 5th of August.
-
-### Wave 2
-
+You can check the official calendar for Batch 7 [here](https://calendar.google.com/calendar/u/1?cid=Y19kMDNlNmE1MzAxNWQxYWMxYjBkNjFkMWI4YzQ0NjUyZTY2ODM3ZjQ3ZDI2YmM5NmM1NGFhYTRhZjdiYWFlMzE5QGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20).
 
 | Activity                          | Date/Deadline                                    |
 |-----------------------------------|--------------------------------------------------|
-| Sign-up on the Admission Portal   | 2022-08-03 00h00 GMT+1 to 2022-08-07  23h59 GMT+1 |
-| Scholarship applications          | 2022-08-03 00h00 GMT+1 to 2022-08-07  23h59 GMT+1 |
-| Test & SLUs                       | 2022-08-08 00h00 GMT+1 to 2022-08-21 23h59 GMT+1 |
-| Candidate Selection               | 2022-08-22                                       |
-| Payments & scholarship interviews | 2022-08-23 00h00 GMT+1 to 2022-08-28 23h59 GMT+1 |
-| Admissions closed                 | 2022-08-29 23h59 GMT+1                           |
+| Sign-up in the Admission Portal   | 09-10-2023 00h00 UTC+1 to 11-10-2023 23h59 UTC+1 |
+| Scholarship applications          | 09-10-2023 00h00 UTC+1 to 22-10-2023 23h59 UTC+1 |
+| Test & SLUs                       | 23-10-2023 00h00 UTC+1 to 05-11-2023 23h59 UTC |
+| Candidate Selection               | 06-11-2023                                       |
+| Payments & scholarship interviews | 07-11-2023 00h00 UTC to 11-11-2023 23h59 UTC|
+| Admissions closed                 | 12-11-2023 23h59 UTC                           |
```

Comment:
YYYY-MM-DD format should be preferred since there is only one known format when starting with YYYY. The DD-MM-YYYY is often times confused with MM-DD-YYYY.

Reply:
Can I use the format 23 Oct 2023?

Reply:
Yes, why not 😄